.circleci/cimodel/data/binary_build_data.py

```diff
@@ -60,7 +60,10 @@ def get_processor_arch_name(cuda_version):
 )
 
 
-DEVTOOLSET_VERSIONS = [3, 7]
+DEVTOOLSET_VERSIONS = [
+    3,
+#    7,  # XXX Temporarily disabled---see https://github.com/pytorch/pytorch/issues/20066
```

Comment:
Can we only remove the cuda 9.0 devtoolset binaries? The others are working fine.